aksel.nav.no/website/sanity/schema/documents/grunnleggende/endringsloggartikkel.tsx

```diff
@@ -0,0 +1,91 @@
+import { defineField, defineType } from "sanity";
+import { FileCodeIcon, FileImageIcon, FileTextIcon } from "@navikt/aksel-icons";
+import { capitalize } from "@/utils";
+import SanityTabGroups from "../presets/groups";
+import { hiddenFields } from "../presets/hidden-fields";
+import { sanitySlug } from "../presets/slug";
+import { titleField } from "../presets/title-field";
+
+export const EndringsloggArtikkel = defineType({
+  title: "Endringsloggartikkel",
+  name: "ds_endringsloggartikkel",
```

Comment:
Since adding `artikkel` already insinuates that there might be other groupings for "endringslogg", would it make sense to change it to `ds_endringslogg_artikkel`?